wadas/ui/configure_ftp_cameras_dialog.py

```diff
@@ -493,18 +493,25 @@ def test_ftp_server(self):
                 self.ui.label_FTPServer_path.text(),
             )
 
+        self.ui.pushButton_testFTPServer.setEnabled(False)
         self.ui.pushButton_stopFTPServer.setEnabled(True)
+        self.ui.buttonBox.setEnabled(False)
```

Comment:
Same as other case, Cancel button shall be enabled. Let's rather capture clicked() evento to trigger fttp server shutdown.